SPECS/softhsm/softhsm.spec

```
@@ -32,28 +27,27 @@ designed to meet the requirements of OpenDNSSEC, but can also work together
 with other cryptographic products because of the PKCS#11 interface.
 
 %package devel
-Summary: Development package of softhsm that includes the header files
-Requires: %{name} = %{version}-%{release}, openssl-devel, sqlite-devel
-%if 0%{?prever:1} || 0%{?prerelease:1}
-BuildRequires: autoconf, libtool, automake
-%endif
+Summary:        Development package of softhsm that includes the header files
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  libtool
```

Thanks for removing `prever` and `prerelease` macros from the spec but I think you left the wrong branch of the `%if` statement. Since both of the macros are not defined, we would actually skip the `BuildRequires`:

```suggestion
Summary:        Development package of softhsm that includes the header files
```